docs/basics/sso/setting-up-sso.md

```diff
@@ -15,6 +15,12 @@ Sauce Labs supports Identity Provider (IdP)-initiated and Service Provider (SP)-
 - An enterprise license for Sauce Labs
 - Organization admin access (see [User Roles](/basics/acct-team-mgmt/managing-user-info) for more information)
 
+## Special cases
```

Comment:
Do you expect to add more exceptions that require additional steps for SAML SSO integration? 
```suggestion
## Special Cases
```

Reply:
not for now, but obviously we will add another cases in the future

Reply:
Set this up so that is works for now, and you can add more cases as needed.